plugin/src/sbt-test/paradox/libraryDependency/build.sbt

```diff
@@ -0,0 +1,6 @@
+lazy val libraryDependencyTest = project
+  .in(file("."))
+  .enablePlugins(ParadoxPlugin)
+  .settings(
+    paradoxTheme := Some(builtinParadoxTheme("generic"))
```

Comment:
Could you use https://jonas.github.io/paradox-material-theme/ for testing please?

Reply:
Sure. Why?

Reply:
I just want to make sure that it's not picking up generic theme by accident.

Reply:
I have changed `check` to also catch the material theme and extended the test to test both the material theme and the generic theme. It fails for the material theme as well, with `assertion failed: pom contains paradox-theme dependency: Some(<artifactId>paradox-material-theme</artifactId>)`.